### DISCUSSION TITLE
Postgresql search does not find exact matches

### DISCUSSION THREAD

**Discussion by doits** [requarks/wiki#3563] [Error / Bug Report]
**Describe the bug**

When I enable postgresql search ("Database - postgresql" at `/a/search`) the search does not find exact matches. For example:

1. Search for `Hom` - it correctly finds the home page

<img width="1088" alt="Bildschirmfoto 2019-07-13 um 20 17 12" src="https://user-images.githubusercontent.com/148003/61175172-3cd39280-a5ab-11e9-8f00-f24dd772645a.png">

2. Search for `Home` - no results

<img width="1085" alt="Bildschirmfoto 2019-07-13 um 20 18 07" src="https://user-images.githubusercontent.com/148003/61175186-5c6abb00-a5ab-11e9-9ad2-f24a48339e6b.png">

(The "Did you mean" suggestion is pointless here, too, since this is my exact search query.)

It seams to be the case for **every possible match**, *not* only page titles. E.g. searching for a word on a page only finds that page when the word is not complete. (E.g. I have the heading `Users` on a page: Searching for `User` finds it, but searching for `Users` not.)

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.0.0-beta.208
 - Database engine: postgres 10.4

**Comment by ethanmdavidson** [comment]
I seem to be having the exact opposite issue. I have a page with title 'Home' which shows up when the search term is 'home' but not when it's 'hom'. This also occurs for text in page bodies.

- OS: CentOS 7.6
- WikiJS version: 2.0.0-beta.303
- Database engine: postgres 11.5
- Search Provider: Database - PostgreSQL

When using the "Database - Basic" search provider, the search works as expected.

**Comment by vabene1111** [comment]
I am having the exakt same problem as descriped by OP.

System: Docker Container (Linux)
WikiJs Version: 2.0.12
PG Version: 11.6
Node JS Version: 10.16.3
Search Provider: Database - PostgreSQL (with german dictionary)

Here are some screenshots for further illustration, i have also tried different capitilitation but that does not change anything.
![2019-12-06_12-33-43-chrome](https://user-images.githubusercontent.com/6819595/70320465-172e7a80-1825-11ea-8944-95c433712e6f.png)
![2019-12-06_12-34-00-chrome](https://user-images.githubusercontent.com/6819595/70320466-172e7a80-1825-11ea-8b2d-17e87942d3cc.png)

**Comment by vabene1111** [comment]
The Specific search shown above in the screenshots works but its still rather buggy overall (both using the simple and the according `german` dict.

![image](https://user-images.githubusercontent.com/6819595/87718392-ed253980-c7b1-11ea-8478-7084b6ead701.png)

![image](https://user-images.githubusercontent.com/6819595/87718418-00d0a000-c7b2-11ea-97b4-324e2dbe2514.png)

![image](https://user-images.githubusercontent.com/6819595/87718439-0a5a0800-c7b2-11ea-8077-ae35de91f8e6.png)

other words, even tough contained completly in the description dont appear at all

![image](https://user-images.githubusercontent.com/6819595/87718460-134ad980-c7b2-11ea-9c2f-6cd464ebc482.png)
